llvm/lib/Transforms/IPO/InputGeneration.cpp

```diff
@@ -130,11 +130,14 @@ bool isLibCGlobal(StringRef Name) {
 }
 
 bool shouldNotStubGV(GlobalVariable &GV) {
-  return isLandingPadType(GV) || isLibCGlobal(GV.getName()) ||
-         StringSwitch<bool>(GV.getName())
-             .Case("llvm.used", true)
-             .Case("llvm.compiler.used", true)
-             .Default(false);
+  if (isLandingPadType(GV) || isLibCGlobal(GV.getName()))
+    return true;
+  else if (GV.getName() == "llvm.used" || GV.getName() == "llvm.compiler.used")
+    return true;
+  else if (GV.getName().starts_with("__llvm") ||
+           GV.getName().starts_with("__prof"))
```

Comment:
@boomanaiden154 Can these be made more specific? If we fail to stub something that is external but is not defined in compiler_rt we will fail the compilation with undefined symbols.

If everything matched here will be defined by the runtime we link in then it's fine